src/cephadm/cephadm

```diff
@@ -3165,25 +3165,29 @@ def command_check_host():
     # caller already checked for docker/podman
     logger.info('podman|docker (%s) is present' % container_path)
 
+    error = ''
```

Comment:
```suggestion
    errors = []
```
this way you can properly join the errors at the end:

```python
'\n'.join(errors)
```

Reply:
:+1: